Examples/timberman.md

```diff
@@ -0,0 +1,72 @@
+# Timberman
+
+**Playstore Link:**  [Timberman](https://play.google.com/store/apps/details?id=com.dm.timber&hl=en)
+
+##### About the game:
```

Comment:
Use the same subheadings as https://raw.githubusercontent.com/GameAutomators/eBook-Source/master/Examples/05-ready-steady-bang.md. You should have **Game Description** here.